src/test/clojure/clara/test_linter.clj

```diff
@@ -0,0 +1,16 @@
+(ns clara.test-linter
+  (:require [clara.rules :refer :all]
+            [clojure.test :refer :all]
+            [clara.rules.testfacts :refer :all]
+            [clara.rules.dsl :as dsl]
+            [clara.tools.testing-utils :refer [assert-ex-data]])
+  (:import [clara.rules.testfacts
+            Cold Hot]))
+
```

Comment:
I'll add tests with more scenarios e.g. more than one duplicate binding and queries if we like this approach.